### ISSUE TITLE
Fix broken links

### ISSUE THREAD

**Issue by liquid-mobius** [somasis/musl-wiki#79]
There are a number of outdated or broken links. Some appear to pose security risk. Please verify all third-party links on the wiki.

**Comment by rofl0r** [comment]
yeah right. why don't you do it and fix it yourself ? PRs welcome.

**Comment by Mobius-code** [comment]
Okay, no problem. I'm not exactly sure how to do that but I'll figure it out. I didn't mean to put it off just on you or anything, it's just that I'm still learning how to use git.

**Comment by richfelker** [comment]
> yeah right. why don't you do it and fix it yourself ? PRs welcome.

That's not a particularly helpful comment. Broken/sus links are a completely valid reportable bug regardless of the reporter's ability ot willingness to do the work fixing them.

**Comment by rofl0r** [comment]
i guess this was addressed by https://github.com/somasis/musl-wiki/commit/62328195cc0a2431c381f02d94eef6a63d7357e5
